### DISCUSSION TITLE
[1551530]Pray Game/祈愿诗篇

### DISCUSSION THREAD

**Discussion by 10YuJi** [pjy612/SteamManifestCache#248] [Share]
appid:
```
1551530
```
keys:
```
1551531;8bf4bb9de5995d70de93ed82c12e20ab74698b9448371dda333a4e7f31e4ad35
1551532;81d1809a02aacdb3d10af04d9637fcea028a71ef0eaf2827af9c07fe12f00998
```
Code:
```
1DA7012E7114C319
```
邮箱:
2504311207@qq.com
[1551530.zip](https://github.com/pjy612/SteamManifestCache/files/13816922/1551530.zip)

内测码变动

**Comment by frankh1111** [comment]
需要去官网下载补丁，包含无码和重要DLC
[祈愿诗篇 – 补丁](https://kaguragamer.com/product/pray-game-patch/)
